AppController/lib/nginx.rb

```diff
@@ -153,6 +153,16 @@ def self.write_fullproxy_app_config(app_name, http_port, https_port,
 JAVA_BLOBSTORE_REDIRECTION
     end
 
+    rest_taskqueue_redirection = <<REST_TASKQUEUE_REDIRECTION
+location ~ /taskqueue/v1beta2/projects/.* {
+    proxy_pass            http://gae_#{app_name}_taskqueue;
```

Comment:
Why do we need an nginx entry for every app, though? I thought the endpoint would be deployment-based.